source/json/highlights.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {},
+  "id": "http://example.com/example.json",
+  "items": {
+    "id": "/items",
+    "properties": {
+      "description": {
+        "id": "/items/properties/description",
+        "type": "string"
+      },
+      "footer": {
```

Comment:
This is new since the schema we discussed as a group, but will allow adding the "See more projects" button (or other bits of HTML as needed).